apps/web/src/components/molecules/CompanyAdListItem/CompanyAdListItem.tsx

```diff
@@ -1,37 +1,77 @@
-import { Badge } from "@dotkomonline/ui"
+import { type JobListing } from "@dotkomonline/types"
+import { Badge, Icon } from "@dotkomonline/ui"
 import { format } from "date-fns"
 import Image from "next/image"
 import { type FC } from "react"
-import { type CareerAd } from "@/api/get-career-ads"
-
 interface CompanyAdListItemProps {
-  career: CareerAd
+  career: JobListing
```

Comment:
Right now we are mixing the use of "CompanyAd", "Career" and "JobListing". In the backend we use "JobListing". I think it would be nice to choose one wording. I suggest changing everything to JobListing, but I am open to other naming suggestions as well.

Reply:
Changed the name on most points